code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -640,3 +640,75 @@
 
 /obj/effect/mob_spawn/human/pirate/gunner
 	rank = "Gunner"
+
+//Forgotten syndicate ship
+
+/obj/effect/mob_spawn/human/syndicatespace
+	name = "Syndicate Ship Crew Member"
+	roundstart = FALSE
+	death = FALSE
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper_s"
+	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
+	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
+	important_info = "Obey orders given by your captain. DO NOT leave the ship."
+	outfit = /datum/outfit/syndicatespace/syndicrew
+
+/datum/outfit/syndicatespace/syndicrew/post_equip(mob/living/carbon/human/H)
+	H.faction |= ROLE_SYNDICATE
```

Comment:
Should this be here, in the outfit datum?

Reply:
Dunno, I was quite literally copying lavaland spawner code.